Pod/Extensions/SSO/SKYAuthContainer+SSO.h

```diff
@@ -0,0 +1,56 @@
+//
+//  SKYAuthContainer+SSO.h
+//  SKYKit
+//
+//  Copyright 2015 Oursky Ltd.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#import <SKYKit/SKYKit.h>
+
```

Comment:
You can remove `_Nonnull` annotations by adding `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`.

Also do this in other `.h` files